src/js/pm-image-editor.js

```diff
@@ -455,6 +474,16 @@
 
                 $scope.updateEditorCss();
             });
+
+            var watchSelectionDimensions = function(value) {
+                if (parseInt(value)) {
```

Comment:
`parseInt` should always be called with it's second parameter (`10`)

See:
* https://stackoverflow.com/questions/6611824/why-do-we-need-to-use-radix
* https://davidwalsh.name/parseint-radix
* https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/parseInt